src/eval-frame/actions/language-actions.js

```diff
@@ -103,75 +103,43 @@ function loadLanguagePlugin(pluginData, dispatch) {
 }
 
 export function evaluateLanguagePlugin(pluginText, evalId) {
-  return dispatch => {
```

Comment:
as above, this is just a bad diff -- the important change is that this is not a thunk, so it doesn't return a function